CHANGELOG.md

```diff
@@ -13,5 +13,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Move config files to standard location based on platform (OSX, Windows, Linux).
 - Align implementation with RFC-002 to use the decision header instead of status codes
+- Upgrade `btsieve` to `url:2.1` 
```

Comment:
I guess I'm the only one who cares about the full stops at the end of these items :stuck_out_tongue: We should be consistent though, so I'll remove them all in another PR.

Reply:
We can do full stops everywhere :)